.github/workflows/push-tests.yml

```diff
@@ -67,6 +67,7 @@ jobs:
   backend-unit-test:
     runs-on: ubuntu-22.04
     timeout-minutes: 30
+    if: contains(github.event.head_commit.modified, 'backend/')
```

Comment:
I like the idea--though we should also run tests if a dependency has been bumped for this container (same goes for all unit tests and their pertinent dependencies)

Reply:
We might need to create a new workflow file to avoid skipping this test in main